esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -973,6 +973,15 @@ void WsWuInfo::getCommon(IEspECLWorkunit &info, unsigned flags)
         info.setDateTimeScheduled(dt.getString(s).str());
 }
 
+void WsWuInfo::setWUAbortTime(IEspECLWorkunit &info, unsigned __int64 abortTS)
+{
+    StringBuffer abortTimeStr;
+    formatStatistic(abortTimeStr, abortTS, SMeasureTimestampUs);
+    abortTimeStr.setCharAt(10, ' ');
+    abortTimeStr.setLength(19);
```

Comment:
I may check length>10 before I set the ' ' and length>19 before set to 19. Not sure I can do more than those.